packages/ppx_make/ppx_make.0.3.4/opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "[@@deriving make]"
+description: "[@@deriving] plugin to generate make functions"
```

Comment:
For future releases may I suggest adding here the example or saying in few more works in which sense? Something like or even slighly more elaborate
```suggestion
description: "[@@deriving] plugin to generate make functions to construct compound types, like records or variants."
```